crates/hir-expand/src/db.rs

```diff
@@ -60,6 +60,7 @@ pub trait ExpandDatabase: RootQueryDb {
     fn proc_macros_for_crate(&self, krate: Crate) -> Option<Arc<CrateProcMacros>>;
 
     #[salsa::invoke(ast_id_map)]
+    #[salsa::lru(512)]
```

Comment:
We should bump this a bit I think maybe to 1024 / 2024. This is keyed over real files and macro expansions, and a lot of macro expansions are pretty small after all. (and this map is still cheaper than `SyntaxNode`s)